out/production/max_EDU/test/com/src/stream_api/StreamApi3.java

```diff
@@ -0,0 +1,25 @@
+package stream_api;
```

Comment:
streamapi or stream.api (a separate folder) would be better names.

Reply:
fixed